app/views/components/description_list/dl.scala.html

```diff
@@ -0,0 +1,21 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(content: Html, classes: Option[String] = None, id: Option[String] = None)
+
+<dl @id.map(i => s"id=$i") @classes.map(c => s"class=$c")>@content</dl>
```

Comment:
Given these four views are all very similar do we think we could refactor them to be one view where we pass in dl or dt or w/e?

Reply:
This is similar to Waigwa's question, answered [here](https://github.com/hmrc/customs-financials-documents-frontend/pull/101/files/bf78cc7fead314f8ba2f402316efbe2d9cddf750#diff-c69d8d6cb8f70a75d217dc11661e07a401f1e2ce3518900a312d40b997383b71)